tests/query/highlights/hack/attribute-type.hack

```diff
@@ -0,0 +1,15 @@
+<<A1>>
+newtype T1 = ?shape(
+//     TODO: ?operator (? not captureable at the moment)
+  ?'int' => int
+//       ^ operator
+);
+
+<<A3(1), A2(2,3,)>>
+//       ^ attribute
+type T2 = (function(T1): string);
+//    ^ type
+//          ^ function (cannot capture keyword "function" as keyword.function)
```

Comment:
Why is that ?

Reply:
(cannot capture keyword "function" as keyword.function as it does not appear in AST). This has to be fixed upstream. The current workaround is to highlight all of `function foo` as `function`

Reply:
This should be changed to https://github.com/slackhq/tree-sitter-hack/blob/dd7c1bb8b11d290db4aec173740efdb510f0bba9/grammar.js#L455 `alias(/function(s)?/, "function")`

Reply:
Might be worth opening a PR then ?

Reply:
https://github.com/slackhq/tree-sitter-hack/pull/36

Reply:
![image](https://user-images.githubusercontent.com/7189118/146015732-8194b768-9d2d-4a39-9612-500a7c991267.png)

This is very creepy: https://github.com/slackhq/tree-sitter-hack/issues/37

Reply:
Great, then I would wait for the PR to be merged, so that we directly pull the changes.

Reply:
I don't think the upstream repo is active anymore.

Reply:
I spoke to some folks at slack and they said they'd look into it, it looks like slackhq/tree-sitter-hack#37 has been fixed